cg/exc.py

```diff
@@ -110,3 +110,9 @@ class ValidationError(CgError):
     """
         Exception related to delivery report validation
     """
+
+
+class StoreError(CgError):
```

Comment:
It can be a slippery slope to introduce custom made exceptions. It is encouraged to use built ins and in some special cases there are suggestions to use. Just mentioning that it can be a good idea to think this through and see if there is a good reason for adding